FogCreek/Glitch-Community
# verbena-moth: Add /pricing page

This is branched off of [classy-afrovenator](https://github.com/FogCreek/Glitch-Community/pull/1155), so maybe we should wait to review/merge this until that PR is merged.

## Links
* [verbena-moth.glitch.me/pricing](https://verbena-moth.glitch.me/pricing)
* [[ch6406]](https://app.clubhouse.io/glitch/story/6406/create-a-pricing-page)

## GIF/Screenshots:
<div>
<img width="325" src="https://user-images.githubusercontent.com/7584833/72162878-ba2c6400-3390-11ea-99b6-c3fc615d240b.png">
<img width="300" src="https://user-images.githubusercontent.com/7584833/72163251-784fed80-3391-11ea-93b4-7d083c42af9d.png" />
</div>

## Changes:
This adds a /pricing page behind the `pufferfish` feature flag.

## How To Test:
*Note that this is the dogfood implementation of this feature, so review should be focused on "Does it work well enough to test internally and iterate on?" rather than polish and cleverness.*

### Testing /pricing Visibility
1. Log in
2. Make sure you're in [@testing-team](https://glitch.com/@testing-team)
3. Make sure the `pufferfish` flag is enabled on [verbena-moth.glitch.me/secret](https://verbena-moth.glitch.me/secret)

| Case  | Expected Outcome | 
| ------------- | ------------- |
| In Testing Team + Enabled the feature on /secret  | /pricing `200` |
| In Testing Team + Disabled the feature on /secret  | /pricing `404` |
| NOT In Testing Team + Enabled the feature on /secret  | /pricing `404`  |
| NOT In Testing Team + Disabled the feature on /secret  | /pricing `404` |

## Feedback I'm looking for:
- Can you create a subscription with test data?
- If you have an active subscription, does the CTA on /pricing say "Manage Your Subscription"?
